cmd/influx/cli/cli.go

```diff
@@ -663,23 +663,24 @@ func (c *CommandLine) Settings() {
 
 func (c *CommandLine) help() {
 	fmt.Println(`Usage:
-        connect <host:port>   connect to another node
-        auth                  prompt for username and password
-        pretty                toggle pretty print
-        use <db_name>         set current databases
-        format <format>       set the output format: json, csv, or column
-        precision <format>    set the timestamp format: h,m,s,ms,u,ns
-        consistency <level>   set write consistency level: any, one, quorum, or all
-        settings              output the current settings for the shell
-        exit                  quit the influx shell
-
-        show databases        show database names
-        show series           show series information
-        show measurements     show measurement information
-        show tag keys         show tag key information
-        show tag values       show tag value information
-
-        a full list of influxql commands can be found at:
+        connect <host:port>   Connect connects to another node specified by host:port
+        auth                  Auth prompts for username and password
+        pretty                Pretty turns on pretty print for the json format	 
+        use <db_name>         Use sets current databases
+        format <format>       Format specifies the format of the server responses: json, csv, or column
+        precision <format>    Precision specifies the format of the timestamp: rfc3339, h, m, s, ms, u or ns
+        consistency <level>   Consistency sets write consistency level: any, one, quorum, or all
+        history               History displays command history
+        settings              Settings outputs the current settings for the shell
+        exit                  Exit quits the influx shell
+
+        show databases        Show database names
+        show series           Show series information
+        show measurements     Show measurement information
+        show tag keys         Show tag key information
+        show tag values       Show tag value information
```

Comment:
this isn't a valid query, it requires `with key = <tag_key>`.

Let's remove it in favor of `show field keys`, which is valid and mirrors `show tag keys`.